vendor/github.com/hashicorp/vault/api/sys_config_cors.go

```diff
@@ -35,63 +35,31 @@ func (c *Sys) CORSStatus() (*CORSResponse, error) {
 	return &result, err
 }
 
-func (c *Sys) ConfigureCORS(req *CORSRequest) (*CORSResponse, error) {
+func (c *Sys) ConfigureCORS(req *CORSRequest) error {
```

Comment:
I'm confused about two things in this file
1) why are we editing in the vendor folder? i'm not totally sure how that's used here, but in my previous experiences the vendor folder was brought in by imports, so do we want to maybe change the source rather than the imported code (if this is true here)? Either way, I'd love to know the rationale here
2) what was going on with all the secrets logic before? I guess why was that there then, and why can it be removed now?

Reply:
This was a change previously made to api that was pulled in when I did "go mod vendor", because the previous commit did not do "go mod vendor" as it should have.  These are not my changes.

For the api and sdk packages, "go mod vendor" just does a copy from api to vendor/github.com/hashicorp/vault/api etc.